consensus/crdt/consensus.go

```diff
@@ -92,6 +93,24 @@ func New(
 
 	ctx, cancel := context.WithCancel(context.Background())
 
+	var blocksDatastore ds.Batching
+	ns := ds.NewKey(cfg.DatastoreNamespace)
+	blocksDatastore = namespace.Wrap(store, ns.ChildString(blocksNs))
+
+	ipfs, err := ipfslite.New(
```

Comment:
ipfslite (bitswap) needs to be created before connecting (bootstrapping) to any other peers. Otherwise nothing works. Thus ipfslite Peer creation is moved here.